pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -85,10 +90,11 @@ func newScanner(integration *storage.ImageIntegration, activeRegistries registri
 	}
 
 	scanner := &scannerv4{
-		name:             integration.GetName(),
-		activeRegistries: activeRegistries,
-		ScanSemaphore:    types.NewSemaphoreWithValue(numConcurrentScans),
-		scannerClient:    c,
+		name:              integration.GetName(),
+		activeRegistries:  activeRegistries,
+		ScanSemaphore:     types.NewSemaphoreWithValue(numConcurrentScans),
+		NodeScanSemaphore: types.NewNodeSemaphoreWithValue(numConcurrentScans), // TODO: Set custom value
```

Comment:
Let's address this TODO now, it does not sound like a lot of work.

Reply:
Checking in Scanner v2, we are using a shared value for both Node Scanning v2 and v2 Image Scans. 
I've removed the TODO markers and propose to follow that logic for Scanner v4 as well.

Reply:
TODO is still there - forgot to push?